docs/scripts/generate_plugin_docs.py

```
@@ -993,29 +1022,93 @@ def generate_all_docs(self, readme_content: str):
+
+
 def main():
-    """Main function to generate plugin documentation."""
+    """Main function to generate plugin and labs documentation."""
     docs_source = os.path.join(os.path.dirname(__file__), "..", "source")
+
     generator = PluginDocGenerator(docs_source)
     readme_url = "https://raw.githubusercontent.com/voxel51/fiftyone-plugins/main/README.md"
-
     try:
         logger.info(f"Fetching README from: {readme_url}")
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)